src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
```

Comment:
not implemented
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/beebbd44-0255-42e7-a5e9-8e4f6432b9b6)